### ISSUE TITLE
Problem with field type "wysihtml"

### ISSUE THREAD

**Issue by ArielBlanco1990** [sachatrauwaen/OpenContent#224]
## Describe the bug
Problem when trying to edit the module when I have a field of type "**wysihtml**". (This worked correctly until OpenContent version 04.07.00)

## Software Versions
DNN: 09.11.02
OpenContent: 05.00.06

## Expected Result
The edit window should be displayed to update the values.

![EditWindow](https://github.com/sachatrauwaen/OpenContent/assets/34067433/3657a70c-00fd-421c-8450-e2300f25b39d)

## Actual Result
When trying to edit the module when I have a field of type "wysihtml". The editing window does not show any fields and the error can be seen in the console.

![OpenContentIssue](https://github.com/sachatrauwaen/OpenContent/assets/34067433/7e67cd43-10b2-4d0c-a6c0-d3fcde0fac96)

![IssueOc](https://github.com/sachatrauwaen/OpenContent/assets/34067433/8b6a3b54-5a27-4433-ba9f-14a6663f51d1)

CC; @WillStrohl

**Comment by WillStrohl** [comment]
@sachatrauwaen Do you think this is something that might be able to be fixed quickly?  Do you need one of us to try and fix it?

**Comment by b-creative** [comment]
Is it An upgraded Open Content? OR a new install?

**Comment by b-creative** [comment]
Also… if you use the fields buttonText and buttonLink… take a look at the field ‘link’. It gives you the possebility to choose what kind of link it is and a text field.

**Comment by ArielBlanco1990** [comment]
@b-creative  The button link thing was just an example. We're really using that kind of value elsewhere.

**Comment by Timo-Breumelhof** [comment]
@b-creative is right, in the new V2 editor WYSIHTML  is not supported any more.

![image](https://github.com/sachatrauwaen/OpenContent/assets/8999382/b0ebb857-9dd2-4d4f-aafa-88b5175e384e)
In that case you would have to replace it with CKEditor.

It should work if you are still using the old form editor.

**Comment by ArielBlanco1990** [comment]
I am using Bootstrap 5 for my template. Any alternative within opencontent with the same possibilities that it provided using WYSIHTML type fields?

**Comment by sachatrauwaen** [comment]
This folder is removed by error 
OpenContent/js/wysihtml/parser_rules
In the DesktopModules folders of dnn installation .
As à workaround you can add it manually from older installation. 
I will fix it for next version.

**Comment by ArielBlanco1990** [comment]
@sachatrauwaen Thank you!

**Comment by Timo-Breumelhof** [comment]
Or you could switch to the CkEditor..

**Comment by WillStrohl** [comment]
> @sachatrauwaen Thank you!

Do you need anything else for this @ArielBlanco1990 ?

**Comment by ArielBlanco1990** [comment]
@WillStrohl No, I think everything is ready on my end. They just have to keep this in mind for future releases.

**Comment by Timo-Breumelhof** [comment]
Has been re-added to the code
